src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -129,11 +133,18 @@ const SignOrExecuteForm = ({
       return
     }
 
+    // For Safe Apps:
+    // - Avoid redirection
+    // - Dispatch an event for return the safeTxHash to the caller
```

Comment:
```suggestion
    // Dispatch a Safe Apps SDK transaction request
```

Reply:
> Dispatch a Safe Apps SDK transaction request

IMO this comment just repeats what the code already says. The original comment is not perfect, but it at least tries to describe why a specific code is needed.

Reply:
In this case I agree with @mikhailxyz. The intention is to make clear we need to avoid the redirection to the transaction history and we need to get the hash for the awaiting caller